aea/protocols/dialogue/base.py

```diff
@@ -1228,9 +1236,10 @@ def is_dialogue_present(self, dialogue_label: DialogueLabel) -> bool:
 
     def get_latest_label(self, dialogue_label: DialogueLabel) -> DialogueLabel:
         """Get latest label for dialogue."""
-        return self._incomplete_to_complete_dialogue_labels.get(
+        complete_dialogue_label = self._incomplete_to_complete_dialogue_labels.get(
             dialogue_label, dialogue_label
         )
+        return complete_dialogue_label if complete_dialogue_label else dialogue_label
```

Comment:
Default is dialogue_label for non-existent keys. The problem here is that there are some keys whose value is None. So:
- If key does not exist, return default dialogue_label
- If key exist but value is None, return dialogue_label
- Otherwise return the correspondent complete_dialogue_label

Reply:
Then why add `None` value [here](https://github.com/valory-xyz/open-aea/pull/125/files#diff-c98ed1e8f81fab5ef743ad9a6572e0f077095a0b1cabbf934eed9183ce0ba044R1134-R1144)? Cannot we simply not add a value for the key?

Reply:
Yes, before we hadn't those dummy keys, but quoting David M: "_by design all incomplete dialogue labels must be present in this mapping_". The existence of those keys is checked in several places so I'm adding _key: None_ for all keys that are going to be checked. Not sure if it's a proper solution but it is the only one I found that fixed all the failing tests.